### ISSUE TITLE
getting 'metadata' error when loading wadors file from dcm4chee server

### ISSUE THREAD

**Issue by NoorUnnisa** [OHIF/Viewers#1751]
**Description**

I am trying to make a generic call to dcm4che server with the loadAndCacheImage method to get the image data and to display it , the details are as follows :

The url am sending is extracted from Ohif study data and is hardcoded here.

```
componentDidMount(){
initCornerStone()


      cornerstoneWadoImageLoader.configure({
        beforeSend: function(xhr) {
            // Add custom headers here (e.g. auth tokens)
            //xhr.setRequestHeader('x-auth-token', 'my auth token');
        },
        useWebWorkers: true,
    });

    cornerstone.enable(document.getElementById('dicomInstanceImage'));

     let url = "wadors:https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs/studies/1.2.826.0.1.3680043.8.1055.1.20111103111148288.98361414.79379639/series/1.2.826.0.1.3680043.8.1055.1.20111103111208937.49685336.24517034/instances/1.2.826.0.1.3680043.8.1055.1.20111103111209352.43625952.05489551/frames/1" 
      
        this.loadAndViewImage(url);

}

```
 ```
loadAndViewImage(imageId) {
      var element = document.getElementById('dicomInstanceImage');
      try {
          var start = new Date().getTime();
          cornerstone.loadAndCacheImage(imageId).then(function(image) {
              debugger
              console.log(image);
              var viewport = cornerstone.getDefaultViewportForImage(element, image);
              
              cornerstone.displayImage(element, image, viewport);
              
          }, function(err) {
              console.error(err);
          });
      }
      catch(err) {
          console.error(err);
      }
```

I am getting the following error in console: can anyone pls help me with it , it seems to be related to setting metadata in cornerstone , how do i set metadata?
index.js:1437 Error: no metadata for imageId wadors:https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs/studies/1.2.826.0.1.3680043.8.1055.1.20111103111148288.98361414.79379639/series/1.2.826.0.1.3680043.8.1055.1.20111103111208937.49685336.24517034/instances/1.2.826.0.1.3680043.8.1055.1.20111103111209352.43625952.05489551/frames/1
    at Ce.promise (cornerstoneWADOImageLoader.min.js:130631)
    at new Promise (<anonymous>)
    at Ce (cornerstoneWADOImageLoader.min.js:130629)
    at loadImageFromImageLoader (cornerstone.js:5177)
    at Module.loadAndCacheImage (cornerstone.js:5242)
    at StudyReport.loadAndViewImage (StudyReport.js:128)
    at StudyReport.componentDidMount (StudyReport.js:119)

**Comment by dannyrb** [comment]
I'm not sure I follow how/why/where you're making this call. Are you able to elaborate?

We recently overhauled our metadata usage to simplify it and make it easier to use as a service by extensions and the viewer application. It could be you're bumping up against an issue caused by that major change.

**Comment by NoorUnnisa** [comment]
I was trying to load and view dicom image as using cornerstone methods in a seperate project by seeing how it is done in ohif codebase, but the same code running smoothly in ohif project and not in my project. I am able to retrieve studies metadata but not the image.
But I realised that Ohif is doing many more thing behind the scenes , so I decided to take up the ohif code for viewer functionality and integrate my requirements in it.
